src/AppBundle/Entity/OrderRepository.php

```diff
@@ -8,26 +8,27 @@
 
 class OrderRepository extends EntityRepository
 {
-    public function getWaitingOrdersForRestaurant(Restaurant $restaurant, \DateTime $date = null)
+    public function getWaitingOrdersForRestaurant(Restaurant $restaurant, \DateTime $now = null)
     {
+        if (null === $now) {
+            $now = new \DateTime();
+        }
+
         $qb = $this->createQueryBuilder('o');
         $qb
             ->join(Delivery::class, 'd', Expr\Join::WITH, 'd.order = o.id')
             ->andWhere('o.restaurant = :restaurant')
             ->andWhere($qb->expr()->in('o.status', [
                 Order::STATUS_WAITING,
-                Order::STATUS_ACCEPTED
+                Order::STATUS_ACCEPTED,
+                Order::STATUS_REFUSED
             ]))
+            ->andWhere('d.date >= :date')
```

Comment:
We should show orders that are "in the past". These are orders have an issue that should be addressed by someone, so they shouldn't disappear in our system. (can happen that the owner is late, courier late, aso.)

Maybe add a label for orders which are late - taking care of the [business assumptions](https://github.com/coopcycle/coopcycle-web/wiki/Business-assumptions)